docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,34 @@
 Version history
 ===============
 
+5.1.1 (2015 April 30)
+---------------------
+
+* Add TIFF writing support to the native C++ implementation
+* Fixed remaining functional differences between Windows and Mac/Linux
+* Improved performance of ImageJ plugin when working with ROIs
+* TIFF export: switch to BigTIFF if .tf2, .tf8, or .btf extensions are used
+* Many bug fixes, including:
+    - fixed upgrade checking to more accurately report when a new version is
+      available
+    - Zeiss CZI
+      - fixed ordering of multiposition data
```

Comment:
This third level of bullet points is not indented enough - compare the entry below.